app/uk/gov/hmrc/nisp/views/account.scala.html

```diff
@@ -52,138 +53,140 @@
     });
 }
 
-@nispMain(serviceInfoContent=Some(includes.loginInfo(user)), userLoggedIn = true, browserTitle = Messages("nisp.main.title"),
-            pageScripts=Some(pageScripts), articleClasses = Some("mainpage"), analyticsAdditionalJs = Some(analyticsAdditionalJs)) {
+@defining(Some(user)) { implicit userOption =>
```

Comment:
After talking with Callum, we're not sure this is the best idea. Maybe we should do the Option conversion in the controller instead.